### DISCUSSION TITLE
github being weird

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#31] [General]
![image](https://user-images.githubusercontent.com/84712013/155673632-e335b4d2-3c26-4cb7-b802-75e1fd4b0a71.png)

**Comment by TechStudent10** [comment]
GitHub isn't being weird. You've just been using it for too long

**Comment by KendallDoesCoding** [comment]
yeah ig this showed up coz i turned on my vpn for a bit for something else.
